ownership-chain/node/src/chain_spec.rs

```diff
@@ -181,7 +181,7 @@ fn testnet_genesis(
 		},
 		collator_selection: laos_ownership_runtime::CollatorSelectionConfig {
 			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
-			candidacy_bond: EXISTENTIAL_DEPOSIT * 16,
+			candidacy_bond: laos_ownership_runtime::ExistentialDeposit::get() + 1,
```

Comment:
my initial comment about this was wrong, this can actually be `Zero::zero()` and doesn't need to be a non-zero value. Assuming that, can we set this to either:

1. bigger explicit value, sth like `UNIT * 16`
2. or to `Zero::zero()` explicitly

I am just suggesting it since it seems this is how other parachains are doing it. At the very least, I am ok with having it simply 1, tying it to `ED` doesn't really make much sense to me though

https://github.com/AstarNetwork/Astar/blob/35f6433780025c7e4012afecb6cb3682ca249bd7/bin/collator/src/parachain/chain_spec/astar.rs#L137

https://github.com/AcalaNetwork/Acala/blob/cab5cd8a8f234a0931bec4a854e805fdb896ffc6/node/service/src/chain_spec/acala.rs#L209

Reply:
sgtm: the zero option is nice and already tested in other blockchain